infra/jenkins-cluster/Dockerfile

```diff
@@ -1,17 +0,0 @@
-FROM jenkinsci/jenkins
```

Comment:
I accidentally pushed this pre-maturely (see https://github.com/google/oss-fuzz/commit/4bfb26e10536e86847f249dd66b8a64bf3bcbfaf)